src/dags/parkeervakken.py

```diff
@@ -369,14 +369,20 @@ def get_modes(row):
         modes.append(tvm_mode)
 
     if any([row.record.BEGINTIJD1, row.record.EINDTIJD1]):
-        x = base.copy()
-        x.update(
-            dict(
-                begin_tijd=format_time(row.record.BEGINTIJD1, datetime.time(0, 0)),
-                eind_tijd=format_time(row.record.EINDTIJD1, datetime.time(23, 59)),
-            )
-        )
-        modes.append(x)
+        begin_tijd = format_time(row.record.BEGINTIJD1, datetime.time(0, 0))
+        eind_tijd = format_time(row.record.EINDTIJD1, datetime.time(23, 59))
+        if begin_tijd < eind_tijd:
```

Comment:
Ik dacht even dat hier strings werden vergeleken, omdat deze functie normaal gesproken juist `parse_time()` heet.

Reply:
Dat is een gooie, `format_time` deed ook formatting, maar niet meer. Ga ik gelijk hernoemen.

Reply:
Done